openmc/mgxs_library.py

```diff
@@ -218,29 +154,28 @@ class XSdata(object):
 
     """
 
-    def __init__(self, name, energy_groups, representation='isotropic'):
+    def __init__(self, name, energy_groups, temperatures=[294.],
```

Comment:
Eh, just room temperature.  Its the same default as in CE mode so I kept that here.

- [x]